lib/beacon/bindings.mli

```diff
@@ -37,6 +37,7 @@ type connection_status = js_string
 type mutez = js_string
 type timeframe = js_string
 type origin_kind = js_string
+type packed_tezos_operation
```

Comment:
This approach (coupled with the `Js.Unsafe.coerce`) seems rather strange to me. And I'm not convinced it's the right approach. I understand that using `Partial` and `Omit` in TypeScript is very strange, and probably fits badly with OCaml's type system, but I think it should be done differently:

For me, `partial_tezos_operation` should be a type that has all the attributes of the intersection of all inherited operations. Then you can build prisms that force data to be present. Since you're usually just "forging" an operation, you can easily be strict about how you construct one.

Reply:
Yes this is better!